dataframe-geo-jupyter/src/main/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/IntegrationGeo.kt

```diff
@@ -54,6 +54,11 @@ internal class IntegrationGeo : JupyterIntegration() {
             useSchema<WithMultiLineStringGeometry>()
         }
 
+        render<GeoDataFrame<*>> {
+            println("GeoDataFrame with ${it.crs?.name?.code} CRS and inner dataframe:")
```

Comment:
It's not ideal, better use `notebook.display("text")` for such things. Reason: we can't guarantee standard stream sync in multithreading envs.